src/terminal.rs

```diff
@@ -82,7 +82,8 @@ fn get_vte_version() -> Option<(u8, u8)> {
 
 /// Checks if the current terminal is WezTerm.
 fn is_wezterm() -> bool {
-    std::env::var("TERM_PROGRAM").map_or(false, |value| value == "WezTerm")
+    std::env::var("TERM").map_or(false, |value| value == "wezterm")
+    | std::env::var("TERM_PROGRAM").map_or(false, |value| value == "WezTerm")
```

Comment:
Please swap the arms; I think we should check `TERM_PROGRAM` first, because it works out of the box in all local sessions.  We only really need to look at `TERM` over SSH sessions, and even then it only works if the user explicitly set it up.